### PR TITLE
fix: automatically merge changes to history back to master

### LINKED ISSUE #7278: chore: add task to port stable history to master branch
The version increment build for stable already adds history to stable branch. It could also open a PR to merge those history.md changes into master branch.

This would allow us to keep the various branches with history.md up to date.

### LINKED ISSUE #7351: chore(common): Update tooling to publish history to both stable and master branches
In Slack comment about #7350

@mcdurdin wrote:
> We need to update the tooling so we publish history to both stable and master branches I think

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #7278.

@keymanapp-test-bot skip

This will need a cherry-pick to beta, once approved, so we get the benefit for stable-16.0.

### CODE REVIEW

File: resources/build/increment-version.sh [hunk 1]
```diff
@@ -156,6 +156,37 @@ if [ "$action" == "commit" ]; then
...
+  # HISTORY.md to master. We don't want to do this for beta or alpha builds;
+  # beta changes will be merged to master periodically anyway.
+  #
+  if [[ "$base" =~ ^stable-[0-9]+\.[0-9]+$ ]]; then
+    git switch master
+
+    # In order to avoid potential git conflicts, we run the history collater
+    # again on the master HISTORY.md. Note that the script always exits 1 to
+    # indicate it hasn't updated VERSION.md. We could tweak that in the future.
+    node resources/build/version/lib/index.js history --no-write-github-comment -t "$GITHUB_TOKEN" -b "$base" || true
```
Comment (darcywong00) [inline resources/build/increment-version.sh:170]: Is `--no-write-github-comment` equivalent to `-write-github-comment false`?

Comment (mcdurdin) [reply resources/build/increment-version.sh:170]: [Yes](https://github.com/yargs/yargs/blob/HEAD/docs/api.md#booleankey).

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.10-alpha](https://keyman.com/downloads/releases/alpha/17.0.10)

**Pull request merged**